paddlenlp/transformers/fnet/modeling.py

```diff
@@ -116,9 +115,9 @@ def forward(self, hidden_states, output_hidden_states=False, return_dict=True):
 
 
 class FNetPooler(Layer):
-    def __init__(self, hidden_size):
+    def __init__(self, config):
```

Comment:
```suggestion
    def __init__(self, config: FNetConfig):
```

在 config 参数上面，建议加上其类型注释，这样在 IDE 当中也会有对应的类型智能提示，从而提升框架的编程体验和代码阅读性。

当然，这个文件中其它相关的构造函数中也建议添加相关的 type hint。